.go-version

```diff
@@ -1 +1 @@
-1.22
+1.21.5
```

Comment:
This change is to make Netlify builds pass as there seem to be failures while installing Go v1.22  and then Go v1.19 is used as a fallback. The minimum version needed to build OPA to Go v1.20